spec/acceptance/place_order_spec.rb

```diff
@@ -126,41 +126,56 @@
 
   context 'given valid item detais' do
     it 'stores the item details' do
-      items_details = [
-        { part_id: '123', part_name: 'Bits', part_price: '5.00', part_quantity: '1'}
-      ]
-      save_items_details.execute(items_details: items_details)
-      items = items_gateway.all
-      expect(items).to eq(items_details)
+      items = [{ 
+        id: '123',
+        name: 'Bits', 
+        price: '5.00',
+        quantity: '1'
+      }]
+      save_items_details.execute(items: items)
+
+      item = items_gateway.all.first
+
+      expect(item[0].id).to eq('123')
+      expect(item[0].name).to eq('Bits')
+      expect(item[0].price).to eq('5.00')
+      expect(item[0].quantity).to eq('1')
     end
 
     it 'stores the multiple items details' do
-      items_details = [
-        { part_id: '233', part_name: 'Bats', part_price: '12.00', part_quantity: '4' },
-        { part_id: '343', part_name: 'Buts', part_price: '17.00', part_quantity: '10' }
+      items = [
+        { id: '233', name: 'Bats', price: '12.00', quantity: '4' },
+        { id: '343', name: 'Buts', price: '17.00', quantity: '10' }
       ]
-      save_items_details.execute(items_details: items_details)
-      items = items_gateway.all
-      expect(items).to eq([
-        { part_id: '233', part_name: 'Bats', part_price: '12.00', part_quantity: '4' },
-        { part_id: '343', part_name: 'Buts', part_price: '17.00', part_quantity: '10' }
-      ])
+
+      save_items_details.execute(items: items)
+
+      item = items_gateway.all.first
```

Comment:
We should probably look at picking out a different variable name for this, but not sure what would be a good pick.

Reply:
I was thinking maybe call it items, then call the items we have assigned above something like order or ordered items.